.github/workflows/ci.yml

```
@@ -69,3 +69,14 @@ jobs:
         with:
           command: strip
           args: --bin cargo-strip -v
+  conclusion:
+    runs-on: ubuntu-latest
+    needs: ci
+    if: always()
+    steps:
+      - name: Result
+        run: |
+          jq -C <<< "${needs}"
+          "$(jq -r 'all(.result as $result | (["success"] | contains([$result])))' <<< "${needs}")"
+        env:
+          needs: ${{ toJson(needs) }}
```

technically, this is not needed. Could simply be

```suggestion
    steps:
      - name: Result
        run: exit 0
```